docs/content/quickstart/build-bundle.md

```diff
@@ -0,0 +1,180 @@
+---
+title: Build your Bundle
+description: Building your first bundle
+---
+
+Once you have a [porter.yaml](porter-yaml) manifest, it's time to build your first bundle. 
+
+The `porter build` command generates this bundle:
+
+```
```

Comment:
When you are demonstrating a command with its output, set the code type to `console` to get the proper highlighting

```